DVFS: CPU: undervolt variable y añadido de frecuencia de operación de 1,64 GHz

Parche basado en https://github.com/armbian/build/blob/master/patch/kernel/sunxi-dev/0012-general-h6-new-opp-table.patch

Signed-off-by: AlexTMjugador <AlexTMjugador@users.noreply.github.com>
---
--- diff -pur linux.orig/arch/arm64/boot/dts/allwinner/sun50i-h6.dtsi	2019-09-15 13:43:29.300026145 +0200
+++ diff -pur linux/arch/arm64/boot/dts/allwinner/sun50i-h6.dtsi	2019-09-15 13:45:23.672026511 +0200
@@ -155,19 +155,25 @@
 
 		opp@1320000000 {
 			opp-hz = /bits/ 64 <1320000000>;
-			opp-microvolt = <900000 900000 1160000>;
+			opp-microvolt = <880000 880000 1160000>;
 			clock-latency-ns = <244144>; /* 8 32k periods */
 		};
 
 		opp@1488000000 {
 			opp-hz = /bits/ 64 <1488000000>;
-			opp-microvolt = <960000 960000 1160000>;
+			opp-microvolt = <940000 940000 1160000>;
 			clock-latency-ns = <244144>; /* 8 32k periods */
  		};
+
+		opp@1640000000 {
+			opp-hz = /bits/ 64 <1640000000>;
+			opp-microvolt = <990000 990000 1160000>;
+			clock-latency-ns = <244144>; /* 8 32k periods */
+		};
 
 		opp@1800000000 {
 			opp-hz = /bits/ 64 <1800000000>;
-			opp-microvolt = <1060000 1060000 1160000>;
+			opp-microvolt = <1050000 1050000 1160000>;
 			clock-latency-ns = <244144>; /* 8 32k periods */
 		};
 	};
